scripts/target_config_lib.lua

```diff
@@ -7,6 +7,8 @@ return function(funcs)
 
 	local target = arg[1]
 	local extra_packages = arg[2]
+	local class_packages_standard = arg[3] or ''
+	local class_packages_tiny = arg[4] or ''
```

Comment:
Make this a table:
```lua
local class_packages = {
  standard = arg[3],
  tiny = arg[4],
}
```
Then the code below can be simplified to
```lua
handle_pkgs(class_packages[dev.options.class])
```
by moving the ` or ''` and `string.gmatch()` into `handle_pkgs` as well.

Reply:
It would be great it we could pass these lists in the environment instead of using separate arguments, but all ideas I have either limit future class names to characters that are valid in shell variable identifiers (disallowing `-` for example) or lead to more Makefile ugliness when generating the environment...

Reply:
I'm not sure how this was resolved either.

Reply:
Faulty push, fixed.